src/app/(nextjs_migration)/(authenticated)/user/settings/page.tsx

```diff
@@ -0,0 +1,288 @@
+
+//       <div class='settings-content'>
+//         <!-- Breach alert preferences -->
+//         <section>
+//           <h3 class='settings-section-title'>
+//             ${getMessage('settings-alert-preferences-title')}
+//           </h3>
+//           ${alertOptions({ csrfToken, allEmailsToPrimary })}
+//         </section>
+
+//         <hr>
+
+//         <!-- Monitored email addresses -->
+//         <section>
+//           <h3 class='settings-section-title'>
+//             ${getMessage('settings-email-list-title')}
+//           </h3>
+//           <p>${getMessage('settings-email-limit-info', { limit })}</p>
+
+//           ${createEmailList(emails, breachCounts)}
+//           <button
+//             class='primary settings-add-email-button'
+//             data-dialog='addEmail'
+//             ${emails.length >= limit ? 'disabled' : ''}
+//           >${getMessage('settings-add-email-button')}</a>
+//         </section>
+
+//         <hr>
+
+//         <!-- Deactivate account -->
+//         <section>
+//           <h3 class='settings-section-title'>
+//             ${getMessage('settings-deactivate-account-title')}
+//           </h3>
+//           <p>${getMessage('settings-deactivate-account-info')}</p>
+//           <a
+//             class='settings-link-fxa'
+//             href='${AppConstants.FXA_SETTINGS_URL}'
+//             target='_blank'
+//           >
+//             ${getMessage('settings-fxa-link-label')}
+//           </a>
+//         </section>
+//       </div>
+//     </div>
+//   `
+// }
+
+// const { allEmailsToPrimary, breachCounts, csrfToken, emails, limit } = data
+
+export default async function Settings() {
+  const l10n = getL10n();
+
+  // const emails = await getUserEmails(req.session.user.id);
+
+  // // Add primary subscriber email to the list
+  // emails.push({
+  //   email: req.session.user.primary_email,
+  //   sha1: req.session.user.primary_sha1,
+  //   primary: true,
+  //   verified: true,
+  // });
+
+  // const breachCounts = new Map();
+
+  // const allBreaches = req.app.locals.breaches;
+  // for (const email of emails) {
+  //   const breaches = await getBreachesForEmail(
+  //     getSha1(email.email),
+  //     allBreaches,
+  //     true
+  //   );
+  //   breachCounts.set(email.email, breaches?.length || 0);
+  // }
+
+  // const {
+  //   all_emails_to_primary: allEmailsToPrimary,
+  //   fxa_profile_json: fxaProfile,
+  // } = req.user;
+
+  // const data = {
+  //   allEmailsToPrimary,
+  //   fxaProfile,
+  //   partial: settings,
+  //   emails,
+  //   breachCounts,
+  //   limit: AppConstants.MAX_NUM_ADDRESSES,
+  //   csrfToken: generateToken(res),
+  //   meta: {
+  //     title: getMessage("settings-meta-title"),
+  //   },
+  //   pathname: req._parsedOriginalUrl?.pathname,
+  // };
+
+  // res.send(mainLayout(data));
+
+  const limit = process.env.MAX_NUM_ADDRESSES;
+  const fxaUrl = process.env.FXA_SETTINGS_URL;
```

Comment:
nit: Should these be `process.env.*` or `AppConstants.*`? (although possibly more of a global question)

```sh
git --no-pager grep -n "AppConstants\." | wc -l # 80
git --no-pager grep -n "process\.env\." | wc -l # 33
```

Reply:
We're kinda leaning towards just using `process.env` from now on - we could possibily later add a separate script that checks for required env vars on startup if we really want to.

Reply:
Related: https://github.com/mozilla/blurts-server/issues/275, and I'd still probably recommend **convict** for managing configs and env vars, etc.

Unrelated, but we should probably also re-triage the 300 open issues and close the majority of them since they're no longer relevant and see which ones we might want to do at some point during the Next migration, or future React migration, or future future TypeScript migration.

Reply:
@pdehaan That's a great call out about looking at current issues on Monitor. I'll put together a issue-triage party for next week.